src/main/java/com/ms/zookeeper/auth/PluggableSASLAuthenticationProvider.java

```diff
@@ -0,0 +1,114 @@
+package com.ms.zookeeper.auth;
```

Comment:
I think we should keep original namespace - org.treadmill, not com.ms.

Reply:
I don't agree. This solution is not Treadmill specific why do we want to use a treadmill namespace? Treadmill is also not an organization.
For example the most well known external java lib is curator made by netflix and it is under com.netflix.curator

Reply:
I agree with @bothejjms here, this is MS plugin, not Treadmill specific.